collectors/charts.d.plugin/ap/metadata.yaml

```diff
@@ -3,52 +3,86 @@ meta:
   module_name: ap
   monitored_instance:
     name: Access Points
-    link: ''
+    link: "https://learn.netdata.cloud/docs/data-collection/networking-stack-and-network-interfaces/linux-access-points"
     categories:
-    - data-collection.networking-stack-and-network-interfaces
-    icon_filename: 'netdata.png'
+      - data-collection.networking-stack-and-network-interfaces
+    icon_filename: "netdata.png"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - ap
+    - access
+    - point
+    - wireless
+    - network
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Maintain surveillance over Access Points, ensuring optimal wireless network connectivity and performance. Monitor and troubleshoot in realtime for high-quality network operations.'
-    method_description: ''
+    metrics_description: "The ap collector visualizes data related to wireless access points."
+    method_description: "It uses the `iw` command line utility to detect access points. For each interface that is of `type AP`, it then runs `iw INTERFACE station dump` and collects statistics."
   supported_platforms:
-    include: []
+    include: [Linux]
     exclude: []
   multi_instance: true
```

Comment:
is it really multi_instance?

Reply:
I believe so. From it's readme, it will:

1. Runs iw dev searching for interfaces that have type AP
2. For each interface found, it runs iw INTERFACE station dump

So if it detects multiple ones, it will do a group of charts for each.